test/integration/smart_answer_flows/another_towing_rules_test.rb

```diff
@@ -0,0 +1,478 @@
+require_relative "../../test_helper"
+require_relative "flow_test_helper"
+
+require "smart_answer_flows/towing-rules"
+
+class TowingRulesTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup do
+    setup_for_testing_flow SmartAnswer::TowingRulesFlow
+  end
+
+  context "start page" do
+    should see_start_page.with_title("  Work out if you’re old enough or have the right kind of licence to tow a trailer from different kinds of vehicle")
+  end
+
+  context "question :towing_vehicle_type" do
+    should see_question(:towing_vehicle_type).with_title("What kind of vehicle do you want to tow with?")
+
+    should "take car or light vehicle users to existing towing entitlements question" do
+      should accept_answer("car-or-light-vehicle").with_next_node(:existing_towing_entitlements?)
+    end
```

Comment:
should "take car or light vehicle users to existing towing entitlements question" do
      should accept_answer("car-or-light-vehicle").with_next_node(:existing_towing_entitlements?)
    end
```suggestion
    should accept_answer("car-or-light-vehicle").with_next_node(:existing_towing_entitlements?)
```

Should's shouldn't be nested - and the description doesn't really tell us anything the assertion doesn't. 

Compare to https://github.com/thoughtbot/shoulda#overview where one-liner shoulds are inside a context.

Reply:
ah, sorry, misunderstood, shoulda be sorted now

Reply:
as a side note, i've seen the word "should" so many times now that it's lost most of its meaning